host/xtest/CMakeLists.txt

```diff
@@ -16,15 +16,20 @@ if(OPENSSL_FOUND)
 	set (OPENSSL_PRIVATE_LINK OpenSSL::Crypto)
 endif()
 
+find_package(Python COMPONENTS Interpreter Development)
+if(NOT Python_Interpreter_FOUND)
+	message(FATAL_ERROR "Python interpreter has not been found")
```

Comment:
Please use the same syntax as the `Threads` test above: `Python interpreter not found`.